project/Dependencies.scala

```diff
@@ -35,7 +35,9 @@ object Dependencies {
   def specs2(scalaVersion: String) =
     scalaVersion match {
       case _ => Seq("org.specs2" %% "specs2-core" % "3.8.6" % "test",
-        "org.specs2" %% "specs2-junit" % "3.8.6" % "test")
+        "org.specs2" %% "specs2-junit" % "3.8.6" % "test",
+        "org.specs2" % "specs2-mock_2.11" % "3.8.6" % "test"
```

Comment:
This does not respect the scala version for the build, so will fail on all but 2.11.